.github/workflows/runTests.yml

```diff
@@ -0,0 +1,64 @@
+name: Tests
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  tests:
+    name: PHP ${{ matrix.php-versions }} on ${{ matrix.os }} w/ ${{ matrix.dependencies }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        php-versions: ['7.3', '7.4']
```

Comment:
I left away 7.2 because I think with the next major release of vfsStream the minimum version should be 7.3 as 7.2 is in its retirement phase. Also left away 8.0 as we don't care about it yet - if we start to aim for that one it can be added.